app/controllers/api/v1/punches_controller.rb

```diff
@@ -0,0 +1,17 @@
+module Api
+  module V1
+    class PunchesController < ActionController::API
+      before_action :authenticate_user!
+
+      def index
+        render json: scopped_punches, status: :ok, each_serializer: PunchSerializer
+      end
+
+      private
+
+      def scopped_punches
```

Comment:
```suggestion
      def scoped_punches
```
Are these supposed to be called `scoped_punches`?